Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -41,6 +41,7 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
             _questionRepository = questionRepository;
             _configuration = configuration;
             _stringConstants = stringConstants;
+            client = new HttpClient();
```

Comment:
There is only one public method which calls a private method to make a post request. The unit test for that public method is not possible since it will throw an error thereby failing the test.